src/components/DeferLoad/index.ts

```diff
@@ -102,6 +102,7 @@ class DeferLoad extends BridgeBase {
           .getActionsApi()
           .showData(this.name, dataContext, dataContextParameters, {
             skipSemanticUrl: true,
+            // @ts-ignore
```

Comment:
Can you leave a comment as to why this ts-ignore is needed so we can pass this back to core team